src/material/card/card.scss

```diff
@@ -11,7 +15,31 @@ $mat-card-default-padding: 16px !default;
 
 @include mdc-helpers.disable-mdc-fallback-declarations {
   // Include all MDC card styles except for color and typography.
-  @include mdc-card.without-ripple($query: mdc-helpers.$mdc-base-styles-query);
+  @include mdc-card.static-styles($query: mdc-helpers.$mdc-base-styles-query);
+}
+
+.mat-mdc-card {
+  @include mdc-helpers.disable-mdc-fallback-declarations {
+    // MDC's theme has `container-elevation` and `container-shadow-color` tokens, but we can't
+    // use them because they output under a `.mdc-card` selector whereas the rest of the theme
+    // isn't under any selector. Even if the mixin is pulled out of the selector, it throws a
+    // different error.
+    @include elevation.elevation(1);
```

Comment:
This could've been written under a `.mat-mdc-card:not(.mat-mdc-card-outlined)` selector, but I wanted to keep the specificity low.